pkg/virt-handler/cache/cache_test.go

```diff
@@ -118,66 +113,60 @@ var _ = Describe("Domain informer", func() {
 
 	Context("with ghost record cache", func() {
 		It("Should be able to retrieve uid", func() {
-			err := AddGhostRecord("test1-namespace", "test1", "somefile1", "1234-1")
+			err := ghostRecordStore.AddGhostRecord("test1-namespace", "test1", "somefile1", "1234-1")
 			Expect(err).ToNot(HaveOccurred())
 
-			uid := LastKnownUIDFromGhostRecordCache("test1-namespace/test1")
+			uid := ghostRecordStore.LastKnownUIDFromGhostRecord("test1-namespace/test1")
 			Expect(string(uid)).To(Equal("1234-1"))
-
 		})
 
 		It("Should find ghost record by socket ", func() {
-			err := AddGhostRecord("test1-namespace", "test1", "somefile1", "1234-1")
+			err := ghostRecordStore.AddGhostRecord("test1-namespace", "test1", "somefile1", "1234-1")
 			Expect(err).ToNot(HaveOccurred())
 
-			record, exists := findGhostRecordBySocket("somefile1")
+			record, exists := ghostRecordStore.findGhostRecordBySocket("somefile1")
```

Comment:
That's a weird mix of public and private functions...

Reply:
I agree, I would like to decouple it mote in the future , exposing only the `getGhostRecords` and filtering can be done on caller.